app/Http/Requests/Api/Auth/RegisterRequest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Http\Requests\Api\Auth;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RegisterRequest extends FormRequest
+{
+    public function rules()
+    {
+        return [
+            'name' => 'required|string|max:255',
+            'email' => 'required|string|email|max:255|unique:users',
+            'password' => 'required|string|min:6|confirmed',
```

Comment:
@aminetiyal Can we add more validations on the password filed when user register like password must include numbers or upper case? for example this regex validating upper case, lower case and numbers `regex:/^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{6,}$/`

Reply:
[commit](https://github.com/BuildForSDGCohort2/Team-248-Backend/pull/29/commits/32624509bb02bddc8f6c1723888c63536335e239)